docs/notes/platforms.md

```diff
@@ -28,4 +28,24 @@ The Mac engine supports:
 *10.7.x (Lion) on Intel
 *10.8.x (Mountain Lion) on Intel
 *10.9.x (Mavericks) on Intel
+*10.10.x (Yosemite) on Intel
+*10.11.x (El Capitan) on Intel
 * **Note:** The engine runs as a 32-bit application regardless of the capabilities of the underlying processor.*
+## iOS
+iOS deployment is possible when running LiveCode IDE on a Mac, and provided Xcode is installed and has been set in LiveCode *Preferences* (in the *Mobile Support* pane).
+Currently, the supported versions of Xcode are:
+*Xcode 4.6 on MacOS X 10.6
```

Comment:
Xcode 4.2 is the last to run on MacOS 10.6 - do we still build iOS engines that can work with that version?